plugin/core/views.py

```diff
@@ -635,7 +637,7 @@ def format_completion(item: dict, index: int, can_resolve_completion_items: bool
         completion = sublime.CompletionItem.command_completion(
             trigger=st_trigger,
             command="lsp_complete_text_edit",
-            args=item,
+            args={"session_name": session_name, "item": item},
```

Comment:
I don't really like this solution because it duplicates a lot of the same data. If there are 200 completion items then every one of them contains the same session_name. Maybe we can store the name in a class-level variable instead, same as the completion items for the LspResolveDocs command?

Reply:
I really prefer to do it correctly (this way) than doing a hack using a global.

Having a global can restrict us in the future if we decide to pull completions from multiple sessions.

Also, I think this is adding relatively small amount of data to already pretty big completion items. We could optimize it by having some unique IDs that would be smaller in size, if it's really an issue.

Reply:
Pushed alternative solution that has smaller footprint but adds extra complexity. Let me know. :)

Reply:
Abort, abort, too much additional code  :)

An alternative solution would be to still use session_name instead of session_id:
![image](https://user-images.githubusercontent.com/22029477/93687818-19667a80-fac1-11ea-9816-dae64b9a61c7.png)

But I still think your original solution with passing the config is better, because it wont cause any future issues,
on the other side I also understand @rwols because this info seems redundant, and we only send the completion request to one session. 

What about this solution here:
![image](https://user-images.githubusercontent.com/22029477/93688090-8713a600-fac3-11ea-9bfd-4ae2161c334b.png)

Reply:
But as said, I also don't mind passing the session name all the way down trough the chain.

Reply:
I have to agree with @predragnikolic. The previous diff was better.

I've been thinking about this. If there's no `"command"` in the completion item we don't have to stash the session name in the args. I think that would keep 95% of completion items untouched.